src/components/action-popover/action-popover.style.js

```diff
@@ -10,11 +9,12 @@ import { isSafari } from "../../__internal__/utils/helpers/browser-type-check";
 const Menu = styled.div`
   ${({ isOpen }) => (isOpen ? "display: block;" : "visibility: hidden;")}
   margin: 0;
-  padding: ${({ theme }) => `${theme.spacing}px 0`};
-  box-shadow: ${({ theme }) => theme.shadows.depth1};
+  padding: var(--spacing100) 0;
+  box-shadow: var(--boxShadow100);
   position: absolute;
-  background-color: ${({ theme }) => theme.colors.white};
-  z-index: ${({ theme }) => `${theme.zIndex.popover}`};
+  background-color: var(--colorsUtilityYang100);
+  z-index: ${({ theme }) =>
+    `${theme.zIndex.popover}`}; // TODO (tokens): implement elevation tokens - FE-4437
```

Comment:
This is a minor thing - personally I think we should be adding reminders like these in the description of the respective ticket instead of with the code. But I'll approve these changes 👍🏼